### PR TITLE
Followup to docs refactor pull request #36067 - fixes gitignore and c…

### PR DESCRIPTION
**Pull request by dharmabumstead**
…leanup in makefile; removes some generated files; moves a straggler to the appropriate subdirectory.

##### SUMMARY

Fixes gitignore plus some straggler issues

##### ISSUE TYPE

 - Feature Pull Request
 - New Module Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,29 +31,29 @@ docs/man/man3/*
...
-docs/docsite/rst/list_of_*.rst
-docs/docsite/rst/module_docs/*.rst
-docs/docsite/rst/modules_by_category.rst
-docs/docsite/rst/network_maintained.rst
-docs/docsite/rst/plugins_by_category.rst
-docs/docsite/rst/partner_maintained.rst
-docs/docsite/rst/playbooks_keywords.rst
+docs/docsite/rst/cli/ansible-*.rst
+docs/docsite/rst/cli/ansible.rst
+docs/docsite/rst/modules/community_maintained.rst
```
Comment (nitzmahone) [inline .gitignore]: Isn't `modules/*.rst` sufficient over all these other specific .rsts in the same dir?

File: docs/docsite/Makefile [hunk 2]
```diff
@@ -69,6 +69,8 @@ clean:
 	-rm rst/config.rst
 	-rm rst/ansible-*.rst
```
Comment (gundalow) [inline docs/docsite/Makefile]: This needs more work, see my comment on your other PR

Most of these other `rm`'s` need updating to add the new paths in

File: docs/docsite/Makefile [hunk 3]
```diff
@@ -69,6 +69,8 @@ clean:
...
 	-rm rst/ansible.rst
+	-rm rst/reference_appendices/config.rst
+	-rm rst/reference_appendices/playbooks_keywords.rst
 
 .PHONEY: docs clean
```
Comment (webknjaz) [inline docs/docsite/Makefile]: I believe, it's a typo: it should be `.PHONY`. Could you please also fix this?

**Review by gundalow — CHANGES_REQUESTED** [review]

File: docs/docsite/Makefile (hunk 3 above)
Comment (webknjaz) [reply docs/docsite/Makefile]: @dharmabumstead

**Review by nitzmahone — CHANGES_REQUESTED** [review]

**Comment by gundalow** [comment]
`./docs/docsite/rst/cli/ansible*` also exists in `./docs/docsite/rst/user_guide/`
Looking at Makefile these are put in `--output-dir=rst/cli/`
Though you have http://54.164.15.46/ansible/user_guide/command_line_tools.html which links to http://54.164.15.46/ansible/user_guide/ansible-playbook.html which seems like a logical place in the TOC

**Pull request merged**